src/plugins/inputs/connect.js

```diff
@@ -1,8 +1,24 @@
+var Msg = require("./models/msg");
```

Comment:
Indeed, see https://github.com/thelounge/lounge/blob/xpaw/auto-rc/src/plugins/inputs/query.js#L3.
Fixed in b9a0ee0.